akka-docs/src/main/paradox/cluster-dc.md

```diff
@@ -48,8 +48,11 @@ top of the Cluster membership information are lost. For example, it wouldn't be
 to use @ref[Distributed Data](distributed-data.md) across the separate clusters.
 
 We often recommend implementing a micro-service as one Akka Cluster. The external API of the
-service would be HTTP or a message broker, and not Akka Remoting or Cluster, but the internal
-communication within the service that is running on several nodes would use ordinary actor 
+service would be HTTP or a message broker, and not Akka Remoting or Cluster (see additional discussion
+ in the Lagom Framework docs: 
+@scala[[Internal and External Commmunication](https://www.lagomframework.com/documentation/latest/scala/InternalAndExternalCommunication.html)]
+@java[[Internal and External Communication](https://www.lagomframework.com/documentation/latest/java/InternalAndExternalCommunication.html)]), 
+but the internal communication within the service that is running on several nodes would use ordinary actor 
```

Comment:
I guess we should use `current` links instead

Reply:
:+1: `current`